fe/src/main/java/org/apache/doris/common/util/RuntimeProfile.java

```diff
@@ -67,19 +73,23 @@ public RuntimeProfile() {
         this.counterMap.put("TotalTime", counterTotalTime);
     }
     
-    public Counter getCounterTotalTime() {
+    public synchronized Counter getCounterTotalTime() {
         return counterTotalTime;
     }
 
-    public Map<String, Counter> getCounterMap() {
+    public synchronized Map<String, Counter> getCounterMap() {
```

Comment:
this 'synchronized' is protect nothing. counterMap is returned, and the caller will get it and do anything without lock protection.